extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/rule/Authorization.scala

```diff
@@ -41,6 +42,7 @@ abstract class Authorization(spark: SparkSession) extends Rule[LogicalPlan] {
 object Authorization {
 
   val KYUUBI_AUTHZ_TAG = TreeNodeTag[Unit]("__KYUUBI_AUTHZ_TAG")
+  var KYUUBI_EXPLAIN_COMMAND_EXECUTION_ID = "org.apache.kyuubi.plugin.spark.authz.util.ReservedKeys"
```

Comment:
Can you search for this class instead? You do not need to rush.

Reply:
Changed..I probably had a brain convulsion at the time. . . . .